src/module/migration/migrations/866-link-to-actor-size-again.ts

```diff
@@ -0,0 +1,30 @@
+import type { ActorPF2e } from "@actor";
+import type { ActorSourcePF2e } from "@actor/data/index.ts";
+import { isActorTypeSizeLinkable } from "@scene/helpers.ts";
+import { MigrationBase } from "../base.ts";
+
+export class Migration866LinkToActorSizeAgain extends MigrationBase {
+    static override version = 0.866;
+
+    override async updateActor(actorSource: ActorSourcePF2e): Promise<void> {
+        if (isActorTypeSizeLinkable(actorSource.type)) return;
+
+        if (actorSource.prototypeToken.flags.pf2e) {
+            actorSource.prototypeToken.flags.pf2e.linkToActorSize = false;
+            actorSource.prototypeToken.flags.pf2e.autoscale = false;
+        }
+    }
+
+    override async updateToken(tokenSource: foundry.documents.TokenSource, actor: ActorPF2e): Promise<void> {
+        if (!actor || !("type" in actor) || isActorTypeSizeLinkable(actor.type)) {
```

Comment:
```suggestion
        if (!actor || isActorTypeSizeLinkable(actor.type)) {
```

Reply:
No longer necessary, you are correct. This was from when I tried to use fromUuid() and never fixed it after.